psh/test-ls-rootfs.py

```diff
@@ -65,6 +65,17 @@ def assert_ls_t(p):
     psh.assert_cmd(p, f'ls -t {ROOT_TEST_DIR}', expected_pattern, msg, is_regex=True)
 
 
+def assert_ls_S(p):
```

Comment:
Idea of different tests for rootfs and non-rootfs targets is all we can do for now, but as it is a common difference between targets maybe some parametrization should be introduced at the .yaml/test_runner level for differentiating between these two.

I bet that some of such splitted tests will be skipped when targets without rootfs will get them (at some point in the future) because it will require for test developer to remember that when the transition occurs such tests should be merged/turned on. And as it is done with `include` in yaml file, each splitted test will require to add new target parameter.

Reply:
Yes, I agree that it could be done automatically in the future.